catalog/models.py

```diff
@@ -1,3 +1,81 @@
+import uuid  # Required for unique book instances
 from django.db import models
+from django.urls import reverse
 
 # Create your models here.
+
+
+class Genre(models.Model):
+    """Model representing a book genre."""
+    name = models.CharField(
+        max_length=200, help_text='Enter a book genre (e.g.Science Fiction)')
```

Comment:
e dùng django I18n để translate text cho nội dung của option `help_text` phục vụ chức năng đa ngôn ngữ của ứng dụng nha
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#internationalization-in-python-code

Reply:
em đã làm